ReactWindows/ReactNative/Bridge/ReactInstance.cs

```diff
@@ -179,13 +179,13 @@ public void Dispose()
             IsDisposed = true;
             _registry.NotifyReactInstanceDispose();
 
-            QueueConfiguration.JavaScriptQueueThread.CallOnQueue(() =>
+            await QueueConfiguration.JavaScriptQueueThread.CallOnQueue(() =>
```

Comment:
Silly question, would introduce a race where we dispose the JS queue thread before we can use it to dispose the bridge.

---
In reply to: [80536408](https://github.com/ReactWindows/react-native-windows/pull/712#discussion_r80536408) [](ancestors = 80536408)